### PR TITLE
Added spark themes

### PR DESCRIPTION
**Pull request by miketamis**

### CODE REVIEW

File: generateThemes.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    } else {
+        primary = _primary;
+        secondary = _secondary;
+    }
+
+    const theme = {
+        "color": {
+          "base": {
+            "primary": {
+              "base": { "value": `{color.brand.${primary}.base.value}` },
```
Comment (SyniaN) [inline generateThemes.js]: So theme.base.value end up being a string `{color.brand.purple.base.value}` for example. How does this string get converted into the actual hex value?

File: generateThemes.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+'purple-green',
+'orange-pink',
+]
+
+
+
+function createTheme(_themeName, inverse) {
+    const themeName = _themeName + (inverse ? '.inverse' : '');
+    console.log(_themeName)
+    const [_primary, _secondary] = _themeName.split('-');
```
[the remark this answers is not in the document]
Comment (miketamis) [reply generateThemes.js]: `_themeName` would stay `purple-orange`, I don't use `themeName` in the split I use `_themeName`

File: generateThemes.js (hunk 1 above)
Comment (miketamis) [reply generateThemes.js]: In the build.js by Style Dictionary https://amzn.github.io/style-dictionary/#/properties?id=attribute-reference-alias

File: generateThemes.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+const fs = require('fs');
+
+const themes = [
+'purple-orange',
```
[the remark this answers is not in the document]
Comment (miketamis) [reply generateThemes.js]: Fixed

File: generateThemes.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (SyniaN) [reply generateThemes.js]: Ooh, tricky lol... Did not see that

**Pull request merged**